src/SignalR/common/Http.Connections/src/HttpConnectionDispatcherOptions.cs

```diff
@@ -108,10 +108,12 @@ public TimeSpan TransportSendTimeout
         get => _transportSendTimeout;
         set
         {
+#pragma warning disable CA1512 // Use ArgumentOutOfRangeException throw helper
```

Comment:
Shouldn't this be fixable now since the APIs exist?

Reply:
I tried this but it looks like they still haven't been added?

Reply:
Something's up with maestro, we haven't had a dependency PR opened here in 4 days. I'll post in FR (CC @dotnet/dnceng)

Reply:
They're investigating (it's a timeout in darc), in the meantime I manually triggered the subscriptions.